internal/app/configurable.go

```diff
@@ -257,6 +256,71 @@ func (app *Configurable) PushToCore(parameters map[string]string) interfaces.App
 	return transform.PushToCoreData
 }
 
+// WrapIntoEvent pushes the provided value as an event to CoreData using the device name and reading name that have been
+// set. If validation is turned on in CoreServices then your deviceName and readingName must exist in the CoreMetadata
+// and be properly registered in EdgeX. This function is a configuration function and returns a function pointer.
```

Comment:
Doesn't `push` the event and validation by core data was removed in 2.0.
```suggestion
// WrapIntoEvent wraps the provided value as an EdgeX Event/Reading using the configured event/reading metadata that have been set. The new Event/Reading is returned to the next pipeline function. This function is a configuration function and returns a function pointer.
```